lib/cryptoexchange/exchanges/bit2c/services/market.rb

```diff
@@ -0,0 +1,36 @@
+module Cryptoexchange::Exchanges
+  module Bit2c
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Bit2c::Market::API_URL}/Exchanges/#{market_pair.base}#{market_pair.target}/Ticker.json"
+        end
+
+        def adapt(output, market_pair)
+          ticker = Cryptoexchange::Models::Ticker.new
+          ticker.base = market_pair.base
+          ticker.target = market_pair.target
+          ticker.market = Bit2c::Market::NAME
+          ticker.ask = NumericHelper.to_d(output['l'])
+          ticker.bid = NumericHelper.to_d(output['h'])
+          ticker.last = NumericHelper.to_d(output['ll'])
+          ticker.volume = NumericHelper.to_d(output['a'])
```

Comment:
@estherleongym
Volume looks like its in `NIS`, we'll need to divide it
```
{"h":26924.26000000,"l":27211.80000000,"ll":26959.61,"a":1.82540265,"av":26829.063687887217650308549732,"c":-2.4369051496399232801360691900}
```
I'll fix it after the merge

Reply:
@estherleongym Ignore my previous email, was looking at the wrong attribute. My bad!